models/models.go

```diff
@@ -12,10 +13,34 @@ import (
 	"github.com/18F/cf-cdn-service-broker/utils"
 )
 
+type State string
+
+const (
+	Provisioning   State = "provisioning"
+	Provisioned          = "provisioned"
+	Deprovisioning       = "deprovisioning"
+	Deprovisioned        = "deprovisioned"
+)
```

Comment:
Minor: Maybe this should go under a `state` package, so that the `const`s are accessed via `state.Provisioning`, etc.